source/contracts/reporting/Market.sol

```diff
@@ -216,7 +214,7 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         for (uint256 j = 0; j < participants.length; j++) {
             _reportingParticipant = participants[j];
             if (_reportingParticipant.getPayoutDistributionHash() == winningPayoutDistributionHash) {
-                require(_reputationToken.transfer(_reportingParticipant, _reportingParticipant.getSize().div(2)));
+                require(_reputationToken.transfer(_reportingParticipant, _reportingParticipant.getSize() / 2));
```

Comment:
For posterity: Dropping safe math because /0 is an exception and we don't have variable divisor anyway.